File: programming files/even_odd_site.cc
#include <iostream>
#include <fstream>
using namespace std;

int main() {
    ofstream file;
    // Open a file to write
    file.open("numbers.html");
    // Define the header for a table in html
    file << "<html>\n<head>\n<title>List of Numbers</title>\n</head>\n<body>\n";
    file << "<table>\n<tr><th>Even Numbers</th><th>Odd Numbers</th></tr>\n";
    // Iterate through numbers <= 50, put evens under even, odds under odds.
    for (int i = 1; i <= 50; i++) {
        if (i % 2 == 0) {
            file << "<tr><td>" << i << "</td><td></td></tr>\n";
        }
        else {
            file << "<tr><td></td><td>" << i << "</td></tr>\n";
        }
    }
    file << "</table>\n</body>\n</html>";
    // Close file.
    file.close();
    // Print the file then close.
    ifstream input("numbers.html");
    cout << input.rdbuf();
    input.close();
    return 0;
}

File: programming files/even_odd_site.py
# opens numbers.html file
with open("numbers.html", "w") as f:
    # writes html header code to the file
    f.write("<html>\n<head>\n<title>List of Numbers</title>\n</head>\n<body>\n")
    # writes html table code to the file
    f.write("<table>\n<tr><th>Even Numbers</th><th>Odd Numbers</th></tr>\n")
    # iterates through every integer 1-50
    for i in range(1, 50):
        # if number is even, it is added left column of the html table
        if i % 2 == 0:
            g.write("<tr><td>{}</td><td></td></tr>\n".format(i))
        # if number is a multiple of 3, it is added to the right column of the html table
        if i % 3 == 0:
            g.write("<tr><td></td><td>{}</td></tr>\n".format(i))
    # end tags for the html code
    f.write("</table>\n</body>\n</html>")
    
# prints the html file
with open("numbers.html"") as f:
    print(f.read())
    
